code/frameworks/nextjs/src/preset.ts

```diff
@@ -170,7 +167,7 @@ export const webpackFinal: StorybookConfig['webpackFinal'] = async (baseConfig,
     configureFastRefresh(baseConfig);
   }
 
-  if (options.features?.experimentalNextRSC) {
+  if (options.features?.experimentalRSC) {
```

Comment:
Should this still be in this file?

Reply:
It controls a webpack configuration. Where would you imagine it living?